packages/mobile/src/dappsExplorer/DAppsExplorerScreen.tsx

```diff
@@ -392,6 +463,12 @@ const styles = StyleSheet.create({
     flex: 1,
     padding: Spacing.Regular16,
   },
+  sectionTitle: {
+    ...fontStyles.label,
+    color: colors.gray4,
+    fontFamily: fontFamily,
```

Comment:
nit: can we avoid redefining `fontFamily` and just use the pre-defined style? wanna be mindful about tweaking specific styles to avoid inconsistencies throughout the app.

Reply:
Yes, I don't know why I did this, I thought the style of this screen didn't match the style-guide but that's not the case. Removed!